src/map/MapNodeSwitchWidget.js

```diff
@@ -209,6 +209,7 @@ export default function Endpoints (props) {
   const [{ apiUrl, groupIndex, hasUrlChanged, isUrlValid }, setApiUrl] = useState(extractUrlState(socket, groups));
   const [storedCustomEndpoints, setStoredCustomEndpoints] = useState(getCustomEndpoints());
 
+  /* eslint-disable no-unused-vars */
```

Comment:
I haven't checked, this is a function. I'm not sure whether it will be needed or not, and out of laziness/low priority didn't cjeck if its affecting anything, so i just removed the warning

Reply:
removed it